src/components/ContentNode/LinksBlock.vue

```diff
@@ -0,0 +1,63 @@
+<!--
+  This source file is part of the Swift.org open source project
+
+  Copyright (c) 2022 Apple Inc. and the Swift project authors
+  Licensed under Apache License v2.0 with Runtime Library Exception
+
+  See https://swift.org/LICENSE.txt for license information
+  See https://swift.org/CONTRIBUTORS.txt for Swift project authors
+-->
+<template>
+  <TopicsLinkCardGrid
```

Comment:
Should we swap the order of this conditional so that it's a little easier to read? Something like `if is {} else {}` as opposed to `if not is {} else {}` like it is here?

Reply:
yep totally.